src/style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.cards {
+  width: 200px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+}
+
+.cards__photo {
+  margin: 0 3px 36px;
+}
+
+.cards__name {
+  margin-bottom: 4px;
+}
+
+.cards__kod {
+  margin-bottom: 16px;
+  font-size: 10px;
+  color: rgba(97, 96, 112, 1);
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.cards__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.cards__price {
+  margin-bottom: 26px;
+  display: flex;
+  align-content: baseline;
+  justify-content: space-between;
+}
+
+.cards__container {
+  margin: 32px 17px 26px;
+}
+
+.cards__photo_product {
+  width: 160px;
+  height: 134px;
+}
+
+.cards__name_title {
+  font-weight: 500;
+  font-size: 12px;
+  color: rgba(6, 11, 53, 1);
+  line-height: 18px;
+}
+
+.cards__rating_comments {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 16px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.cards__price_name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(97, 96, 112, 1);
+}
+
+.cards__price_numb {
+  font-weight: 700;
+  font-size: 16px;
+  color: rgba(6, 11, 53, 1);
+  line-height: 18px;
+}
+
+.cards__button {
+  box-sizing: border-box;
+  width: 100%;
+  height: 40px;
+  border: 1px solid rgba(0, 172, 220);
+  border-radius: 5px;
+  text-transform: uppercase;
+  background-color: rgb(0, 172, 220);
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-decoration: none;
+  padding: 10px 55px;
+}
+
+.cards__button:hover {
+  background-color: #fff;
+  color: rgba(0, 172, 220);
+  border: 1px solid rgba(0, 172, 220);
+}
+
+.stars {
+  display: flex;
+  font-size: 10px;
+  font-weight: 400;
+  color: rgba(6, 11, 53, 1);
+}
```

Comment:
You should create separate css file for each BEM block (one for card, one for stars) and reset.css, normalize.css for setting main default styles